nemo/core/optim/optimizers.py

```diff
@@ -64,6 +66,14 @@
     except (ImportError, ModuleNotFoundError):
         HAVE_APEX_DISTRIBUTED_ADAM = False
 
+    try:
+        # Try importing wrapper for Apex FusedAdam optimizer
+        from nemo.core.optim.megatron_fused_adam import MegatronFusedAdam
+
+        AVAILABLE_OPTIMIZERS['megatron_fused_adam'] = MegatronFusedAdam
+    except (ImportError, ModuleNotFoundError):
+        logging.warning("Could not import MegatronFusedAdam optimizer")
```

Comment:
Remove this warning. It just pollutes the import of NeMo with unnecessary log messages

Reply:
Removed